Scripts/script-CommonServerPython.yml

```diff
@@ -1234,4 +1256,4 @@ dependson: {}
 timeout: 0s
 tests:
   - TestPYCommonServer
-releaseNotes: "Add djb2 hash function"
```

Comment:
Not sure this should be removed

Reply:
It's not my release note, i think this goes to Alex after every merge (it's there from the last merge of this file to Master).

Not sure though, how can we know for sure?

Reply:
These releaseNotes are staying there until Itay removes them after the release, thus these release notes are there for the next release, you can check with him to make sure of that

Reply:
@ronykoz is right. In this cases, just add numbering and append your release notes.